pytest_fixtures/core/contenthosts.py

```diff
@@ -214,7 +220,8 @@ def module_container_contenthost(request, module_target_sat, module_org, module_
         "rhel_version": "8",
         "distro": "rhel",
         "no_containers": True,
-        "network": "ipv6" if settings.server.is_ipv6 else "ipv4",
+        # TODO(vsedmik): Check whether this is valid for dualstack scenaro
+        "network": "ipv6" if module_target_sat.network_type == NetworkType.IPV6 else "ipv4",
```

Comment:
Could we go directly with host attr?
```suggestion
        # TODO(vsedmik): Check whether this is valid for dualstack scenaro
        "network": settings.content_host.attributes.network_type,
```

We will need to parametrize this (and flatpak) fixture soon with RHEL 9 and 10.1 like we do with `rhel_contenthost`s. But that's definitely out of scope of this PR.

Reply:
We can, but it will have its consequences in the dual-stack pipeline. If that setting is equal to `dualstack` in the dual-stack pipeline, the content host will be deployed with a dual network stack—IPv4 and IPv6. But in case of dual-stack testing, we've decided not to deploy dual-stack hosts on the client end but parameterize the tests that test clients so that they run twice - they spin up an IPv4 host and then an IPv6 host.

I will leave it up to you what behavior you want in your tests.